cmd/landscaper-controller/app/app.go

```diff
@@ -23,6 +23,7 @@ import (
 	mockctlr "github.com/gardener/landscaper/pkg/deployer/mock"
 	executionactuator "github.com/gardener/landscaper/pkg/landscaper/controllers/execution"
 	installationsactuator "github.com/gardener/landscaper/pkg/landscaper/controllers/installations"
+	"github.com/gardener/landscaper/pkg/landscaper/webhook"
```

Comment:
```suggestion
	landscaperwebhook "github.com/gardener/landscaper/pkg/landscaper/webhook"
```
maybe add a import alias to be more specific what webhook is added.